lib/command/contact/details.php

```diff
@@ -63,12 +63,12 @@ public function __construct( Entity\Contact_Id $contact_id ) {
 	 *
 	 * @return Entity\Contact_Id
 	 */
-	public function get_contact_id(): Entity\Contact_Id {
+	private function get_contact_id(): Entity\Contact_Id {
 		return $this->contact_id;
 	}
 
 	/**
-	 * {@inheritDoc}
+	 * @internal
```

Comment:
> It won't stay hidden with `{@inheritDoc}` ?

Nope, it was like that before (locally), and it didn't work.
The result was literally the ``{@inheritDoc}`` text.

The reason for that is because if we add `@internal` to the parent method, then there won't be a documentation for it.. Thus, using `{@inheritDoc}` won't work.